Src/Base/AMReX_FabArrayCommI.H

```diff
@@ -240,6 +243,8 @@ FabArray<FAB>::FillBoundary_finish ()
     fbd.reset();
 
 #endif
+
+    BL_PROFILE_SYNC_STOP();
```

Comment:
It might too late because the function might return early.

Reply:
Should be fixed.